config/v1/types_oauth.go

```diff
@@ -51,6 +51,16 @@ type TokenConfig struct {
 	// accessTokenInactivityTimeoutSeconds - DEPRECATED: setting this field has no effect.
 	// +optional
 	AccessTokenInactivityTimeoutSeconds int32 `json:"accessTokenInactivityTimeoutSeconds,omitempty"`
+
+	// accessTokenInactivityTimeout defines the default token
+	// inactivity timeout for tokens granted by any client.
+	// The value represents the maximum amount of time that can occur between
+	// consecutive uses of the token. Tokens become invalid if they are not
+	// used within this temporal window. The user will need to acquire a new
+	// token to regain access once a token times out.
+	// Takes valid time duration string such as "5m", "1.5h" or "2h45m".
+	// +optional
```

Comment:
tell me what happens if this is not set.

Reply:
added explanation on the behavior.